src/js/components/Footer/stories/Social.js

```diff
@@ -33,13 +33,14 @@ export const Social = () => (
   // Uncomment <Grommet> lines when using outside of storybook
   // <Grommet theme={...}>
   <Box>
-    <Main background="light-2" elevation="large" pad="medium" gap="large">
+    <Main background="light-1" elevation="large" pad="medium" gap="large">
       <Text margin="small" size="xsmall">
         Main Content
       </Text>
       <Box flex />
     </Main>
-    <Footer background="light-4" pad="small">
+    {/* Choosing Color similar to brand color for text "Gormmet" */}
```

Comment:
```suggestion
```
Can we remove this comment

Reply:
Hey, I removed this comment in my latest commit